bin/runYANGgenericstats.sh

```diff
@@ -129,212 +110,101 @@ if [ "$IS_PROD" = "True" ]; then
    done
 
    date +"%c: forking all sub-processes for OpenROADM versions" >>$LOG
-   declare -a PIDS2
-   running=0
    for path in $(ls -d $TMP/openroadm-public/*/); do
-      ((running = running + 1))
       version=$(basename $path)
-      (python $BIN/yangGeneric.py --metadata "OpenRoadm $version: YANG Data Models compilation from https://github.com/OpenROADM/OpenROADM_MSA_Public/tree/master/model" --lint --prefix OpenROADM$version --rootdir "$TMP/openroadm-public/$version/" >>$LOG 2>&1) &
-      PIDS2+=("$!")
-      wait_for_processes "${PIDS2[@]}"
-   done
-   # Wait for all child-processes
-   for PID in ${PIDS2[@]}; do
-      wait $PID || exit 1
+      yang_generic --metadata "OpenRoadm $version: YANG Data Models compilation from https://github.com/OpenROADM/OpenROADM_MSA_Public/tree/master/model" --lint --prefix OpenROADM$version --rootdir "$TMP/openroadm-public/$version/"
```

Comment:
`--metadata` should contain Github URL, please replace `master` with name of the branch, for example 5.1.0